examples/pdc_test_cache.rb

```diff
@@ -46,3 +46,4 @@ def main
 end
 
 main if __FILE__ == $PROGRAM_NAME
+puts    "dfdfdfdfdfdf"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.